bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/visibility/TargetVisibilityCalc.scala

```diff
@@ -73,7 +73,7 @@ object TargetVisibilityCalc {
     Key(A, GN, Ngs) -> hr( 1.0 -> Bad,     (4.0  + EQUALITY_LIMIT) -> Limited, ( 7.0 + EQUALITY_LIMIT) -> Good,    (22.0 + EQUALITY_LIMIT) -> Limited),
     Key(A, GN, Lgs) -> hr( 0.0 -> Bad,     (5.0  + EQUALITY_LIMIT) -> Limited, ( 8.0 + EQUALITY_LIMIT) -> Good,     21.0 -> Limited),
     Key(A, GS, Ngs) -> hr( 2.0 -> Bad,     (5.0  + EQUALITY_LIMIT) -> Limited, ( 7.0 + EQUALITY_LIMIT) -> Good,    (23.0 + EQUALITY_LIMIT) -> Limited),
-    Key(A, GS, Lgs) -> hr( 1.0 -> Bad,     (6.0  + EQUALITY_LIMIT) -> Limited, ( 8.0 + EQUALITY_LIMIT) -> Good,    (22.0 + EQUALITY_LIMIT) -> Limited),
+    Key(A, GS, Lgs) -> hr(20.0 -> Bad,     (6.0  + EQUALITY_LIMIT) -> Limited, ( 7.0 + EQUALITY_LIMIT) -> Good,    (18.0 + EQUALITY_LIMIT) -> Limited),
```

Comment:
I don't remember how this works.  Switching from 1.0 to 20.0 seems like a large change?

Reply:
It is a very large change 🤷